fili-core/src/main/java/com/yahoo/bard/webservice/application/AbstractBinderFactory.java

```diff
@@ -754,6 +755,7 @@ protected final void setupDruidDimensionsLoader(
                 throw new IllegalStateException(e);
             }
         } else if (BardFeatureFlag.DRUID_CACHE.isOn()) {
+            LOG.warn("Cache V1 has been deprecated and is no longer supported. Cache V2 is the default cache now.");
```

Comment:
Instead we should say something like, this feature flag is deprecated, use the new configuration parameter to set desired caching strategy.